client/systems.go

```diff
@@ -138,3 +139,27 @@ func (client *Client) RebootToSystem(systemLabel, mode string) error {
 	}
 	return nil
 }
+
+type SystemDetailsData struct {
```

Comment:
this name probably doesn't need Data at the end

Reply:
I removed this